backends/qualcomm/quantizer/utils.py

```diff
@@ -123,11 +132,11 @@ def annotate_binary(node: Node, quantization_config: QuantizationConfig) -> None
 
     input_qspec_map = {}
     input_act0 = node.args[0]
-    if isinstance(input_act0, Node):
+    if isinstance(input_act0, Node) and _is_input_float_tensor(input_act0):
```

Comment:
nit: we could also move `isinstance(input_act0, Node)` check into `_is_input_float_tensor` function as well